app/pods/bucket-type/template.hbs

```diff
@@ -145,15 +145,17 @@
           <tr>
             <td class="key">Stored buckets</td>
             <td class="value">
-              <ul class='button-list small'>
-                {{#each model.bucketList.buckets as |bucket|}}
-                  <li>
-                    {{link.link-bucket bucket=bucket}}
-                  </li>
-                {{else}}
-                  No buckets found
-                {{/each}}
-              </ul>
+              {{#pagination-component chunk-length=1 total-length=model.bucketList.total section-request='retrieveRequestedBuckets'}}
```

Comment:
I noticed that the bucketlist model has two properties, `total` and `count`. What is the difference? With my seed data these two values are different, with `total` reflecting all list items. I'm not sure what `count` represents.